lib/archethic/account/mem_tables_loader.ex

```diff
@@ -130,6 +130,9 @@ defmodule Archethic.Account.MemTablesLoader do
 
       {{to, _token_address}, utxo} ->
         TokenLedger.add_unspent_output(to, utxo, timestamp)
+
+      {{to, _token_address, _token_id}, utxo} ->
```

Comment:
Why do you have the two cases ?

Reply:
we use this two different cases in aggregating the movements. A map is used to aggregate , where key is 
`to , address, tokenid`  for nft 
and f
` to, token_address` for fungible tokens.
Bug was reported by Prince, it fixes that.